data/base-spells.yaml

```diff
@@ -326,6 +327,11 @@ spell_data:
     abbrev: AA
     mana: 1
     recast: 1
+  Athleticism:
+    skill: Augmentation
+    abbrev: ATH
```

Comment:
Technically `ATH` isn't the abbreviation, it's just the first few letters of the spell's name. Could you replace this with `Athleticism`?